inc/sai.h

```diff
@@ -77,7 +79,7 @@ typedef enum _sai_api_t
     SAI_API_VIRTUAL_ROUTER   =  5, /**< sai_virtual_router_api_t */
     SAI_API_ROUTE            =  6, /**< sai_route_api_t */
     SAI_API_NEXT_HOP         =  7, /**< sai_next_hop_api_t */
-    SAI_API_NEXT_HOP_GROUP   =  8, /**< sai_next_hop_group_api_t */
+    SAI_API_GROUP            =  8, /**< sai_group_api_t */
```

Comment:
A better descriptive name would be better. Say ENDPOINT_GROUP or EGRESS_GROUP etc.